lighthouse-cli/run.js

```diff
@@ -57,7 +57,7 @@ function parseChromeFlags(flags = '') {
 function getDebuggableChrome(flags) {
   return ChromeLauncher.launch({
     port: flags.port,
-    chromeFlags: parseChromeFlags(flags.chromeFlags),
+    chromeFlags: parseChromeFlags(flags.chromeFlags + ' --enable-features=NetworkServiceInProcess'),
```

Comment:
nit: should we do this on the post parse-side?

`.concat('--enable-features=NetworkServiceInProcess')` feels a bit safer somehow

Reply:
does this override any other `--enable-features`, or are they combined